lib/Controller/LogoutController.php

```diff
@@ -60,7 +66,10 @@ public function logout(string $userId): JSONResponse {
 			);
 		}
 
-		$this->userSession->setUser($user);
+		$impersonatee = $this->userManager->get($userId);
```

Comment:
I love the more clearer distinction on the name of the Getter function on the event.
...tee vs ...tor is visually so similar. `$impersonatedUser` would be more different.

Reply:
Same gut feeling, :+1:

Reply:
Ah, actually as i did in the Event class… =D